sdk/eventhub/azure-eventhubs/azure/eventhub/common.py

```diff
@@ -349,11 +356,19 @@ def _set_partition_key(self, value):
             self.message.annotations = annotations
             self.message.header = header
 
+    @property
+    def size(self):
+        """The size of EventDataBatch object in bytes
+
+        :rtype: int
+        """
+        return self._size
+
     def try_add(self, event_data):
         """
-        The message size is a sum up of body, properties, header, etc.
-        :param event_data: ~azure.eventhub.EventData
-        :return: None
+        Try to add an EventData object, the size of EventData is a sum up of body, application_properties, etc.
```

Comment:
Broken
![image](https://user-images.githubusercontent.com/1050156/68139993-f3071300-fedf-11e9-9cb9-62b3ef03f973.png)

Reply:
it is fixed